packages/ui-kit/package.json

```diff
@@ -17,7 +17,8 @@
   "module": "dist/commercetools-docs-ui-kit.esm.js",
   "files": ["dist", "package.json", "LICENSE", "README.md"],
   "scripts": {
-    "generate-icons": "svgr --out-dir src/icons/generated --typescript -- src/icons/svg"
+    "generate-icons": "svgr --out-dir src/icons/generated --typescript --icon \"src/icons/svg/*.svg\" \"src/icons/lord-icon/*.svg\" && yarn copy-lord-icons-json",
+    "copy-lord-icons-json": "cpy --cwd=src 'icons/lord-icon/*.json' ../dist/icons/lord-icon"
```

Comment:
Is the custom CLI used for windows compatibility?  I would be OK having the kit not support windows based developmemt without a Unix shell.  We have quite a lot of unix shell scripting, even in the docs repo.